utils/rpms/daos.spec

```
@@ -233,8 +233,14 @@ PREFIX="%{?_prefix}"
 mkdir -p %{?buildroot}/%{_sysconfdir}/ld.so.conf.d/
 echo "%{_libdir}/daos_srv" > %{?buildroot}/%{_sysconfdir}/ld.so.conf.d/daos.conf
 mkdir -p %{?buildroot}/%{_unitdir}
+echo "rhel %{?rhel}"
+%if (0%{?rhel} <= 7)
```

I think you want:

```suggestion
%if (0%{?rhel} >= 7)
```
here, to do the next two lines on EL7 (and higher).  But really, you probably need to assess the situation on EL8 and see if this is needed there and if not:
```suggestion
%if (0%{?rhel} = 7)
```

---

Ok trying to do an exact match against EL7 and otherwise use the newer versions.